README.md

```diff
@@ -1,68 +1,34 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Welcome to my Portfolio
 
-## Available Scripts
+This project was made using React and Elastic UI as a learning practice.
```

Comment:
Make React and Elastic UI links to the projects

Reply:
Updated